src/Checkbox/web/Checkbox.js

```diff
@@ -85,6 +93,7 @@ Checkbox.propTypes = {
   onChange: PropTypes.func,
   onBlur: PropTypes.func,
   error: PropTypes.string,
+  truncate: PropTypes.bool,
```

Comment:
The checkbox shouldn't accept the truncate prop, we'll have to find an alternate way to implement this.